includes/REST/ReverseWithdrawalController.php

```diff
@@ -103,6 +103,7 @@ public function register_routes() {
                     'methods'             => WP_REST_Server::READABLE,
                     'callback'            => [ $this, 'get_transaction_types' ],
                     'permission_callback' => [ $this, 'get_transaction_types_permissions_check' ],
+                    'args'                => $this->get_collection_params(), // It returns a collection of items, can added pagination type support
```

Comment:
No argument is used by the `get_transaction_types` method.

Reply:
No argument is used on `get_transaction_types` but $request is passed on `format_collection_response` as a param and there they used,

```
 $per_page = ! empty( $request['per_page'] ) ? (int) $request['per_page'] : -1;
 $page     = ! empty( $request['page'] ) ? (int) $request['page'] : 1;
 $request->get_query_params();
```

So, I added this `'args' => $this->get_collection_params()` .